README.md

```diff
@@ -1,221 +1,10 @@
 # Matrix-appservice-slack
-A Matrix &lt;--> Slack bridge
+A Matrix &lt;--&gt; Slack bridge
```

Comment:
We should lay out what features we support here, as a high level description. This probably means specifying that we support webhooks and "rich" bot bridging, as well as puppeting. Might also be nice to have a one liner explaining that you need a homeserver to proceed any further and all that faff.